lib/color.js

```diff
@@ -219,6 +218,75 @@ class ColorXY {
         return kelvinToMireds(kelvin);
     }
 
+    /**
```

Comment:
@sjorge Continuing on 

> interactions with scene_recall, we should probably do a syncColorState there too right?

Yes

Reply:
OK, I'll add the calls there too on the next pass.